src/v2/models/AppState.js

```diff
@@ -217,8 +220,15 @@ export default Model.extend({
       && messagesObjs.value.some(messagesObj => messagesObj.i18n?.key.startsWith(keySubStr));
   },
 
+  clearPrevsiousData() {
```

Comment:
nit: 
```suggestion
  clearAppStateCache() {
```

Reply:
It's not only about cacheing but actual data. Will use `cache` be confusing?